third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -68,6 +68,11 @@ JVM_KOTLINC_OPTS = [
     "-Xno-use-ir",
 ]
 
+# Default deps
+DEFAULT_DEPS = [
+    "//third_party/java/jsr250_annotations",
```

Comment:
Add a target to  `java/arcs/sdk/BUILD`. The name of the target should appear to refer to the "Generated" annotation, but it _actually_ is an alias to third party (now that I think of it, it should be exactly the alias in this line). 
Replace the value of `DEFAULT_DEPS` with the new target in the SDK.

Reply:
For Wasm compatibility: add an _actual_ build rule for the Wasm annotation. It should be the same name as the target described above, but with a `-wasm` suffix.

Reply:
After looking at the rest of the PR -- the same pattern should apply for `-js`, too.